config/releases.exs

```diff
@@ -283,3 +283,38 @@ config :service_acquire, Acquire.Application,
   ]
 
 config :service_acquire, Acquire.Db.Presto, presto: Keyword.put(presto_db, :user, "acquire")
+
+# SERVICE DEFINE
+config :service_define, DefineWeb.Endpoint,
+  http: [:inet6, port: String.to_integer(System.get_env("DEFINE_PORT") || "4005")],
```

Comment:
```suggestion
  http: [:inet6, port: String.to_integer(System.get_env("PORT") || "4005")],
```
This'll be consistent with all the other services. `PORT` is set at runtime in Kubernetes, so they can all use the same variable and it's less to coordinate in Helm.